src/containers/trends/TrendsSVGContainer.js

```diff
@@ -0,0 +1,220 @@
+    if (this.props.showingCbg) {
+      return (
+        <CBGSlicesAnimationContainer
+          data={this.props.data}
+          focusedSlice={this.props.focusedSlice}
+          focusSlice={this.props.focusSlice}
+          margins={this.props.margins}
+          svgDimensions={{ height, width }}
+          tooltipLeftThreshold={this.props.tooltipLeftThreshold}
+          unfocusSlice={this.props.unfocusSlice}
+          xScale={this.props.xScale}
+          yScale={this.props.yScale}
+        />
+      );
+    }
+    return null;
+  }
+
+  renderSmbg() {
+    if (this.props.showingSmbg) {
+      const rangeOverlay = this.props.smbgRangeOverlay ?
+        (<SMBGRangeAvgAnimationContainer
+          data={this.props.data}
+          focusRange={this.props.focusRange}
+          smbgRangeOverlay={this.props.smbgRangeOverlay}
+          tooltipLeftThreshold={this.props.tooltipLeftThreshold}
+          unfocusRange={this.props.unfocusRange}
+          xScale={this.props.xScale}
+          yScale={this.props.yScale}
+        />) : null;
+      return (
+        <g id="smbgTrends">
+          {[rangeOverlay]}
+        </g>
+      );
+    }
+    return null;
+  }
+
+  render() {
+    const { containerHeight: height, containerWidth: width } = this.props;
+    return (
+      <svg height={height} width={width}>
+        <BackgroundWithTargetRange
+          bgBounds={this.props.bgBounds}
+          linesAtThreeHrs
+          margins={this.props.margins}
+          smbgOpts={this.props.smbgOpts}
+          svgDimensions={{ height, width }}
+          xScale={this.props.xScale}
+          yScale={this.props.yScale}
+        />
+        <XAxisLabels
+          margins={this.props.margins}
+          useRangeLabels={false}
+          xScale={this.props.xScale}
+        />
+        <XAxisTicks
+          margins={this.props.margins}
+          xScale={this.props.xScale}
+        />
+        <YAxisLabelsAndTicks
+          bgBounds={this.props.bgBounds}
+          bgUnits={this.props.bgUnits}
+          margins={this.props.margins}
+          yScale={this.props.yScale}
+        />
+        {this.renderCbg()}
+        {this.renderSmbg()}
+        <TargetRangeLines
+          bgBounds={this.props.bgBounds}
+          smbgOpts={this.props.smbgOpts}
+          xScale={this.props.xScale}
+          yScale={this.props.yScale}
+        />
+      </svg>
+    );
+  }
+}
+
+TrendsSVGContainer.defaultProps = {
+  margins: MARGINS,
+  smbgOpts: SMBG_OPTS,
+  // for time values after 6 p.m. (1800), float the tooltips left instead of right
+  tooltipLeftThreshold: 6 * THREE_HRS,
+};
+
+TrendsSVGContainer.propTypes = {
+  bgBounds: PropTypes.shape({
+    veryHighThreshold: PropTypes.number.isRequired,
+    targetUpperBound: PropTypes.number.isRequired,
+    targetLowerBound: PropTypes.number.isRequired,
+    veryLowThreshold: PropTypes.number.isRequired,
+  }).isRequired,
+  bgUnits: PropTypes.oneOf(['mg/dL', 'mmol/L']).isRequired,
```

Comment:
No, this is just a new component that I'd started before we merged together our work. I've fixed.